crates/metal-ungram/metal.ungram

```diff
@@ -75,9 +75,9 @@ Type =
   | RefType
   | BinaryType
 
-NameType = Name generics:NameTypeMaybeGenerics?
-NameTypeMaybeGenerics = '[' generics:NameTypeGenerics ']'
-NameTypeGenerics = Type*
+NameType = Name generics:NameTypeGenerics?
+NameTypeGenerics = '[' generics:NameTypeGenericsInner ']'
+NameTypeGenericsInner = Type* // FIXME
```

Comment:
what needs fixing here?

Reply:
missing commas. the grammar basically says `HashMap[Key Value]` instead of `HashMap[Key, Value]`